spec/requests/organized/ambassador_dashboards_request_spec.rb

```diff
@@ -34,28 +33,66 @@
 
     describe "show" do
       it "renders the ambassador dashboard" do
-        get base_url
+        FactoryBot.create_list(:ambassador_task, 2)
+        FactoryBot.create_list(:ambassador, 2, organization: organization)
+
+        get organization_ambassador_dashboard_path(organization)
+
         expect(response.status).to eq(200)
-        expect(assigns(:ambassadors).count).to eq(1)
+        expect(assigns(:ambassadors).count).to eq(3)
         expect(response).to render_template(:show)
       end
     end
 
     describe "resources" do
       it "renders the ambassador resources" do
-        get "#{base_url}/resources"
+        get resources_organization_ambassador_dashboard_path(organization)
```

Comment:
Why switch to routes instead of explicit paths?

Reply:
The benefits I see to using explicit paths is that it adds one extra bit of test coverage.

I can see an argument for using paths instead, in that we generally use them in the code.

Reply:
1. In the preexisting tests, for consistency with the new tests
2. In the new tests, because url helpers to provide a layer of indirection so that we're not propagating knowledge of the routes throughout the app. (Suppose, for example, we want to refactor underscores to dashes in routes for the sake of SEO.)

I'm not sure I see where the added test coverage would come from, but I'll go ahead ship and iterate since this is a bug fix. Happy to standardize on something and refactor for uniformity.

Reply:
The "extra test coverage" is a little bit of a stretch, because I'm not sure that it actually is useful test coverage (like shoulda matchers) - but by writing out the route as a URL string, we get [routing specs](https://relishapp.com/rspec/rspec-rails/v/3-8/docs/routing-specs).

There are [two routing specs](https://github.com/bikeindex/bike_index/tree/master/spec/routing), both for complicated routing parameters, both were used as unit tests when I was verifying things worked as expected; if we switched fully over to request specs and wrote out URLs, these tests wouldn't be necessary.

Merging this was definitely the right choice. I'm adding additional explanation to the request spec migration issue #845